numpy/_core/src/common/numpyos.c

```diff
@@ -416,7 +416,7 @@ NumPyOS_ascii_isupper(char c)
  * Same as tolower under C locale
  */
 NPY_NO_EXPORT int
-NumPyOS_ascii_tolower(int c)
+NumPyOS_ascii_tolower(char c)
```

Comment:
The definition is correct, it is meant to replace `int tolower(int)`. Please change the declaration.

Reply:
Thanks Matti, updated the declaration.